tools/topology/topology2/platform/intel/bt-generic.conf

```diff
@@ -160,36 +190,47 @@ Object.Pipeline {
 				copier_type	"SSP"
 				stream_name	$BT_NAME
 				node_type	$I2S_LINK_INPUT_CLASS
-				num_input_audio_formats 4
-				num_output_audio_formats 4
```

Comment:
@ranj063 may I removed too many things but can we agree that this is completely wrong? There are 3 formats supported